src/shared/seccomp-util.c

```diff
@@ -1112,6 +1112,17 @@ int seccomp_memory_deny_write_execute(void) {
                 r = seccomp_rule_add_exact(
                                 seccomp,
                                 SCMP_ACT_ERRNO(EPERM),
+                                SCMP_SYS(mmap2),
```

Comment:
Hm, I've just `grep -r mmap2 | grep SYSCALL_DEFINE`ed. The result:
```
arch/tile/kernel/sys.c:SYSCALL_DEFINE6(mmap2, unsigned long, addr, unsigned long, len,
arch/mips/kernel/syscall.c:SYSCALL_DEFINE6(mips_mmap2, unsigned long, addr, unsigned long, len,
arch/mips/kernel/linux32.c:SYSCALL_DEFINE6(32_mmap2, unsigned long, addr, unsigned long, len,
arch/microblaze/kernel/sys_microblaze.c:SYSCALL_DEFINE6(mmap2, unsigned long, addr, unsigned long, len,
arch/s390/kernel/compat_linux.c:COMPAT_SYSCALL_DEFINE1(s390_mmap2, struct mmap_arg_struct_emu31 __user *, arg)
arch/s390/kernel/sys_s390.c:SYSCALL_DEFINE1(mmap2, struct s390_mmap_arg_struct __user *, arg)
```
Seems like `MemoryDenyWriteExecute` doesn't make sense on `s390` at least